data/net/minecraft/client/gui/Gui.mapping

```diff
@@ -30,42 +39,102 @@ CLASS net/minecraft/client/gui/Gui
 		ARG 1 mountEntity
 	METHOD getVisibleVehicleHeartRows (I)I
 		ARG 1 mountHealth
+	METHOD guessChatUUID (Lnet/minecraft/network/chat/Component;)Ljava/util/UUID;
+		COMMENT Try to find the UUID of player that send message
+		ARG 1 message
+	METHOD handleChat (Lnet/minecraft/network/chat/ChatType;Lnet/minecraft/network/chat/Component;Ljava/util/UUID;)V
+		ARG 1 chatType
+		ARG 2 message
+		ARG 3 sender
+	METHOD lambda$displayScoreboardSidebar$1 (Lnet/minecraft/world/scores/Score;)Z
+		ARG 0 score
 	METHOD onDisconnected ()V
 		COMMENT Reset the GuiPlayerTabOverlay's message header and footer
 	METHOD render (Lcom/mojang/blaze3d/vertex/PoseStack;F)V
-		ARG 1 matrixStack
-		ARG 2 partialTicks
+		ARG 1 poseStack
+		ARG 2 partialTick
 	METHOD renderCrosshair (Lcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 1 matrixStack
+		ARG 1 poseStack
 	METHOD renderDemoOverlay (Lcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 1 matrixStack
+		ARG 1 poseStack
 	METHOD renderEffects (Lcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 1 matrixStack
+		ARG 1 poseStack
 	METHOD renderExperienceBar (Lcom/mojang/blaze3d/vertex/PoseStack;I)V
-		ARG 1 matrixStack
-		ARG 2 xPos
+		ARG 1 poseStack
+		ARG 2 x
+	METHOD renderHeart (Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/gui/Gui$HeartType;IIIZZ)V
+		ARG 1 poseStack
+		ARG 2 heartType
+		ARG 3 x
+		ARG 4 y
+		ARG 5 vOffset
+		ARG 6 blink
+		ARG 7 halfHeart
+	METHOD renderHearts (Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/world/entity/player/Player;IIIIFIIIZ)V
+		ARG 1 poseStack
+		ARG 2 player
+		ARG 3 x
+		ARG 4 y
+		ARG 5 rowHeight
+		ARG 6 regen
+		ARG 7 healthMax
+		ARG 8 health
+		ARG 9 healthLast
+		ARG 10 absorb
+		ARG 11 blink
 	METHOD renderHotbar (FLcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 1 partialTicks
-		ARG 2 matrixStack
+		ARG 1 partialTick
+		ARG 2 poseStack
 	METHOD renderJumpMeter (Lcom/mojang/blaze3d/vertex/PoseStack;I)V
-		ARG 1 matrixStack
-		ARG 2 xPosition
+		ARG 1 poseStack
+		ARG 2 x
 	METHOD renderPlayerHealth (Lcom/mojang/blaze3d/vertex/PoseStack;)V
-		ARG 1 matrixStack
+		ARG 1 poseStack
 	METHOD renderPortalOverlay (F)V
-		ARG 1 timeInPortal
+		ARG 1 portalTime
```

Comment:
`timeInPortal` reads better.

Reply:
I change it to be consistent with LocalPlayer.portalTime, but I can change it back.